Lullabot/media_mpx
# Player formatter

This PR:

* Cleans up some code debt in notifications with some refactoring of the NotificationQueuer into a second NotificationListener class.
* Supports adding a player formatter for videos.
* Stores the account public IDs on the account entities to save some complexity in rendering players.

To test:

* Re-save any mpx account config entities to ensure they have a public ID saved.
* In the field display options, set the mpx URL to the `mpx player` formatter.
* View a video!

Note that players may not be 100% functional if they require additional javascript that is not provided by the player iframe. For this PR, I consider "success" to be "we have an iframe with a rendered player inside of it".
